src/fanuc_sim/description/scene.xml

```diff
@@ -82,6 +83,7 @@
     <!-- Define the block with AprilTags on each face -->
     <body name="block1" pos="0.2 0.75 0.575">
       <freejoint name="block1" />
+      <site name="block1" pos="0 0 0" />
```

Comment:
are these sites necessary for the weld constraint to activate?

Reply:
Yes